config/components/keydb.json

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "keydb"
```

Comment:
As I mentioned in the description, there's no CPE in NVD associated to this name. Please note KeyDB is a Redis fork, so I'm wondering whether it makes sense to use `redis` instead.

The same happens with `valkey`: no CPE in NVD. That said, we're using `valkey` in the config.

cc @gongomgra

Reply:
I agree that it makes sense to use `redis` in these early stages of the fork, but we should keep an eye to detect new CVEs only affecting `keydb` assets and not `redis` once the fork's code has diverged further. I would also only use `redis` now in case there is any CVE from `redis` we know also affects `keydb`, to avoid populating the `data/keydb` folder with information that will never be matched (OSV ingestion process will be happy as well). The same for `valkey`

Reply:
Thanks! Check my proposal at my last commit